src/main/java/ai/rev/speechtotext/StreamingClient.java

```diff
@@ -202,6 +202,14 @@ private String buildURL(
         urlBuilder.addQueryParameter(
             "delete_after_seconds", String.valueOf(sessionConfig.getDeleteAfterSeconds()));
       }
+      if (sessionConfig.getStartTs() != null) {
+        urlBuilder.addQueryParameter(
+                "start_ts", String.valueOf(sessionConfig.getStartTs()));
+      }
```

Comment:
are we using 2-space indents? If so
```suggestion
        urlBuilder.addQueryParameter(
          "start_ts", String.valueOf(sessionConfig.getStartTs()));
      }
```